common/base/src/logging.cpp

```diff
@@ -27,66 +29,67 @@ namespace genomeworks
 
 namespace logging
 {
-static std::shared_ptr<spdlog::logger> logger = nullptr;
+static std::unique_ptr<std::ostream> out_stream_ = nullptr;
+static std::unique_ptr<std::ofstream> out_file_  = nullptr;
 
-LoggingStatus Init(const char* filename)
+static LogLevel level_ = LogLevel::error;
+
+void check_logger()
 {
-    // for now, first call wins:
-    if (logger != nullptr)
-        return LoggingStatus::success;
+    if (out_stream_ == nullptr)
+    {
+        std::cerr << "GenomeWorks logger not initialized yet. Initializing default logger now." << std::endl;
+        initialize_logger(LogLevel::error);
+    }
+}
+
+std::string log_level_str(LogLevel level)
```

Comment:
Hehe yes, I do like this idea but I just wanted to remove spdlog with a simple logging mechanism. I think we have this now, but this would be a great next step. I've added these as future upgrades. https://github.com/clara-parabricks/GenomeWorks/issues/616